eval/src/vespa/eval/instruction/generic_create.cpp

```diff
@@ -81,21 +81,22 @@ void my_generic_create_op(State &state, uint64_t param_in) {
                                                          param.dense_subspace_size,
                                                          param.my_spec.size());
     std::vector<vespalib::stringref> sparse_addr;
-    for (const auto & kv : param.my_spec) {
-        sparse_addr.clear();
-        for (const auto & label : kv.first) {
-            sparse_addr.emplace_back(label);
-        }
-        T *dst = builder->add_subspace(sparse_addr).begin();
-        for (size_t stack_idx : kv.second) {
-            if (stack_idx == CreateParam::npos) {
-                *dst++ = T{};
-            } else {
-                const Value &child = state.peek(stack_idx);
-                *dst++ = child.as_double();
+    param.my_spec.each_entry([&](const auto &key, const auto &values)
+        {
+            sparse_addr.clear();
+            for (const auto & label : key) {
+                sparse_addr.emplace_back(label);
```

Comment:
I think this was problematic with some compilers since we need to use the implicit cast from string to stringref (there is no stringref constructor taking a string).

prefer push_back here.